benchmarks/many_modules/benchmarker.rb

```diff
@@ -54,6 +57,19 @@ def generate
         }, f)
       end
 
+      File.open(File.join(locales, 'config.yaml'), 'w') do |f|
+        f.puts(
+          {"gettext"=>
+            {"project_name"=>"module#{i}",
```

Comment:
Part of the issue we saw may be that metadata name did not match project_name. We've addressed that but haven't tested performance with it yet. So if these names don't match we might see worse performance over many runs.